DummyServer/ClientApp/src/app/home/home.component.html

```diff
@@ -0,0 +1,220 @@
+<p>This is the control panel for the dummy server. To get started,</p>
+<ul>
+  <li>Start and configure BuildNotifications with the <strong>dummy plugin</strong></li>
+  <li>Add dummy <strong>user, definitions</strong> and <strong>branches</strong></li>
+  <li>Create builds based on these mock data and start developing with BuildNotifications</li>
+</ul>
+<p>Duplicate values are <code>forbidden.</code>Values are persisted on the device the server is hosted on.</p>
+
+<div class="row">
+  <div class="col">
+    <div class="card">
+      <div class="card-header">
+        <div class="icon-big icon-branch" style="float: left"></div>
+        <div class="mx-5">
+          <code style="color: #5C6982">
```

Comment:
I don't care for the Dummy Server but in a "real world" app I would say to remove most of - if not all - inline styles and use css classes for this.

Something like `h3 { color: #5C6982 }` in the home.component.scss file

Reply:
But I don't want every h3 to be this color. I want the styling of the <code> tag, but in a different color than default. I'm not sure how to do this in css?

Reply:
Good question. I doubt this is possible in plain css. And my scss-foo is not strong enough for this :D

Reply:
Aight. Then I'll leave this for now